core/src/replay_stage.rs

```diff
@@ -99,7 +99,6 @@ const MAX_VOTE_SIGNATURES: usize = 200;
 const MAX_VOTE_REFRESH_INTERVAL_MILLIS: usize = 5000;
 // Expect this number to be small enough to minimize thread pool overhead while large enough
 // to be able to replay all active forks at the same time in most cases.
-const MAX_CONCURRENT_FORKS_TO_REPLAY: usize = 4;
```

Comment:
The default is still to have concurrent fork replay disabled. However, this bit of guidance is lost if someone does want to enable concurrent replay but is unsure about a good value to pick. So, I was a little conflicted on how to handle this.

After typing this out, one thing I could do is undelete this constant and use the value as an upper bound for  `--replay-forks-threads` (instead of num threads). If we go that route, I think we should keep the constant here and make it public, that way the constant is "closer to the relevant code". Thoughts ?

Reply:
Yeah, using this as an upper bound sounds reasonable to me.

Reply:
Tweaked the comment slightly but went towards using this value as upper bound